src/lib/services/proxy-service.ts

```diff
@@ -109,6 +110,7 @@ export class ProxyService {
         if (!client) {
             client = this.createClientForProxyToken(token);
             this.clients.set(token.secret, client);
+            this.config.eventBus.emit(PROXY_REPOSITORY_CREATED, {});
```

Comment:
Is the second argument needed?

Reply:
I didn't have it there initially, but I couldn't find example of code that didn't provide something. And I wondered if that had anything to do with snapshot suddenly failing. But I'll remove it